src/vaadin-message-input-text-area.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { TextAreaElement } from '@vaadin/vaadin-text-field/src/vaadin-text-area.js';
+import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+
+registerStyles(
+  'vaadin-message-input-text-area',
+  css`
+    :host {
+      align-self: stretch;
+      flex-grow: 1;
+      padding: 0;
+    }
+  `,
+  { moduleId: 'vaadin-message-input-text-area-styles' }
+);
+
+/**
+ * The text area element for a message input.
+ *
+ * ### Styling
+ *
+ * See [`<vaadin-text-area>` documentation](https://github.com/vaadin/vaadin-text-field/blob/master/src/vaadin-text-area.js)
+ * for `<vaadin-message-input-text-area>` parts and available slots
+ *
+ * See [ThemableMixin – how to apply styles for shadow parts](https://github.com/vaadin/vaadin-themable-mixin#readme)
+ *
+ * @extends TextAreaElement
+ */
+class MessageInputTextAreaElement extends TextAreaElement {
+  static get is() {
+    return 'vaadin-message-input-text-area';
+  }
+}
```

Comment:
This component could contain `ready` block from `vaadin-message-input` and configure some defaults:

https://github.com/vaadin/vaadin-messages/blob/beef43f8c81ca076a16f4295571dc185a41d642f/src/vaadin-message-input.js#L112-L130

Reply:
`this.__submit()` is part of the `vaadin-message-input.js` file. Do I need to dispatch an event in `vaadin-message-input-text-area`, then catch it in `vaadin-message-input`, and call `__submit();`?

Reply:
I did just that. Done.